apiserver/facades/agent/uniter/status.go

```diff
@@ -75,5 +95,113 @@ func (s *StatusAPI) UnitStatus(args params.Entities) (params.StatusResults, erro
 // ApplicationStatus returns the status of the Applications and its workloads
 // if the given unit is the leader.
 func (s *StatusAPI) ApplicationStatus(args params.Entities) (params.ApplicationStatusResults, error) {
-	return s.applicationGetter.Status(args)
+	result := params.ApplicationStatusResults{
```

Comment:
so this was moved from ApplicationStatusGetter.Status without changes?
not easy to read the diff here

Reply:
Well, the change was to get the application status from the cache, and the unit status values from the new method that only takes one query.